### PR TITLE
[AutoScheduler] Print the time used for measurement

### PR DESCRIPTION
**Pull request by merrymercy**

### CODE REVIEW

File: src/auto_scheduler/utils.h [hunk 1]
```diff
@@ -253,6 +254,16 @@ inline std::string Chars(const char& str, int times) {
   return ret.str();
 }
 
+/*! \brief Print the time elapsed */
+inline void PrintTimeElapsed(std::chrono::time_point<std::chrono::high_resolution_clock> t_begin,
```
Comment (FrozenGene) [inline src/auto_scheduler/utils.h:258]: I think it is a little strange to pass the `t_begin` and elapse the duration time. I would suggest we pass the `duration` to `PrintElapsedTime` (this name should be better, but current fn name is ok too).

i.e. 
```cpp
double duration = std::chrono::duration_cast<std::chrono::duration<double>>(
                        std::chrono::high_resolution_clock::now() - t_begin)
                        .count();
inline void PrintElapsedTime(double duration, int verbose) {
  StdCout(verbose) << "Time elapsed: " << std::fixed << std::setprecision(2) << duration << " s"
                   << std::endl;
}
```

Comment (merrymercy) [reply src/auto_scheduler/utils.h:258]: I feel that
`
double duration = std::chrono::duration_cast<std::chrono::duration<double>>(
                        std::chrono::high_resolution_clock::now() - t_begin)
                        .count();
`
is too long and makes the readbility worse. This is the reason for creating `PrintElapsedTime`

Comment (FrozenGene) [reply src/auto_scheduler/utils.h:258]: How about we define one macro to help the readability?

```cpp
#define duration(x) std::chrono::duration_cast<std::chrono::duration<double>>(x).count()
#define now() std::chrono::high_resolution_clock::now()

auto elapsed_time = duration(now() - t_begin)

PrintElapsedTime(elapsed_time, verbose);

inline void PrintElapsedTime(double elapsed_time, int verbose) {
  StdCout(verbose) << "Time elapsed: " << std::fixed << std::setprecision(2) << elapsed_time << " s"
                   << std::endl;
}
```

**Review by comaniac — APPROVED** [review]
LGTM. A miner suggestion is now we will have many "Time elapsed" on the console, so it might be confusing sometimes, although I know we have separators for each step. Maybe consider passing a prefix to the print function?

**Pull request merged**